.gitignore

```diff
@@ -1,5 +1,14 @@
-node_modules
-**/host_vars/iojs-*
-**/host_vars/node-*
-**/host_vars/test-*
-**/host_vars/release-*
+# tools/jenkins-status:
+node_modules/
```

Comment:
Non-blocking could this be `/node_modules/`? We might one day want to vendor in a packaged tool

Reply:
It could be `/tools/jenkins-status/node_modules/`, but I think defaulting to not vendoring node_modules is a better default, then people can change it if they want to include.